.github/workflows/bot.yml

```diff
@@ -12,9 +12,9 @@ on:
     types:
       - created
       - edited
-  pull_request:
-    types:
-      - opened
+  # pull_request:
+  #   types:
+  #     - opened
```

Comment:
@0xcodercrane this threw an error when I opened the pull request. Why is this here?

Reply:
@0xcodercrane